deploy/kubernetes/base/node_windows/node.yaml

```diff
@@ -24,7 +24,7 @@ spec:
         kubernetes.io/os: windows
       containers:
         - name: csi-driver-registrar
-          image: gke.gcr.io/k8s-staging-csi/csi-node-driver-registrar-win
+          image: k8s.gcr.io/sig-storage/csi-node-driver-registrar
```

Comment:
Can you explain why we don't have to use a windows image for the registrar? Do we have a multi-arch build or something like that?

(and ditto for the driver image)

Reply:
right, both container images support multi-arch build now.